lib/kitchen/driver/azurerm.rb

```diff
@@ -336,7 +340,13 @@ def create(state)
           # Retrieve the public IP from the resource group:
           result = get_public_ip(state[:azure_resource_group_name], "publicip")
           info "IP Address is: #{result.ip_address} [#{result.dns_settings.fqdn}]"
-          state[:hostname] = result.ip_address
```

Comment:
Thank you for your contribution!

Maybe we can rework this code to have a default then if we have the value set then we use the hostname? Something like this?

```ruby
state[:hostname] = result.ip_address
if config[:use_fqdn_hostname]
  info "Using FQDN to communicate instead of IP"
  state[:hostname] = result.dns_settings.fqdn
end
```

Reply:
Okay, ill refactor that in 👍

Reply:
Ive added that in